src/map/instance.c

```diff
@@ -806,11 +826,12 @@ static void do_reload_instance(void)
 				break;
 		}
 
-		if( k != instance->list[i].num_map ) /* any (or all) of them were disabled, we destroy */
+		if( k != instance->list[i].num_map ) /* any (or all) of them were disabled, we destroy */ {
```

Comment:
should be ``if (k != instance->list[i].num_map) /* any (or all) of them were disabled, we destroy */ {``

Reply:
ah - sorry forgot that I should be updating the styles when I copy/paste